easybuild/easyconfigs/c/cram/cram-0.7-GCCcore-11.2.0.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'cram'
+version = '0.7'
+
+homepage = 'https://bitheap.org/cram'
+description = "Cram is a functional testing framework for command line applications."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['7da7445af2ce15b90aad5ec4792f857cef5786d71f14377e9eb994d8b8337f2f']
+
+builddependencies = [('binutils', '2.31.1')]
```

we should use the binutils version that `GCC/11.2.0` uses
```suggestion
builddependencies = [('binutils', '2.37')]
```